app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -247,9 +247,10 @@ class Settings private constructor(
         appContext.getPreferenceKey(R.string.pref_key_tracking_protection_custom_cookies),
         true
     )
+
     val blockCookiesSelectionInCustomTrackingProtection by stringPreference(
         appContext.getPreferenceKey(R.string.pref_key_tracking_protection_custom_cookies_select),
-        ""
+        "all"
```

Comment:
"all" is to aggressive to be the default we should have "social" -> "Cross-site and Social media trackers" as default following the same order as the spinner

Reply:
Also I noticed in desktop if we switch policies like from **Custom** to **Strict** or **Standard** the custom values return to the default values, I think we need to include this functionality, as the user could get a lot of site breakage if we keep the previous values as the user could think that is our default values

Reply:
> Also I noticed in desktop if we switch policies like from **Custom** to **Strict** or **Standard** the custom values return to the default values, I think we need to include this functionality, as the user could get a lot of site breakage if we keep the previous values as the user could think that is our default values

We've discussed this and this behaviour remains TBD. Thanks @Amejia481 😄

Reply:
👍